### ISSUE TITLE
Error when we would like create new user

### ISSUE THREAD

**Issue by LinkPhoenix** [Ylianst/MeshCentral#4]
Hello,
after installing Meshcentral2 and creating the first user with administrative rights it is impossible to create a new user. There seems to be a problem with the "meshuser.js" script at line 369

we have this error after trying to create a new user

```
/root/meshcentral/node_modules/meshcentral/meshuser.js:369
                            obj.parent.hash(command.pass, function (err, salt, hash) {
                                       ^

TypeError: obj.parent.hash is not a function
    at WebSocket.<anonymous> (/root/meshcentral/node_modules/meshcentral/meshuser.js:369:40)
    at emitTwo (events.js:126:13)
    at WebSocket.emit (events.js:214:7)
    at Receiver.ontext (/root/meshcentral/node_modules/express-ws/node_modules/ws/lib/WebSocket.js:841:10)
    at /root/meshcentral/node_modules/express-ws/node_modules/ws/lib/Receiver.js:536:18
    at /root/meshcentral/node_modules/express-ws/node_modules/ws/lib/Receiver.js:368:7
    at /root/meshcentral/node_modules/express-ws/node_modules/ws/lib/PerMessageDeflate.js:249:5
    at afterWrite (_stream_writable.js:454:3)
    at onwrite (_stream_writable.js:445:7)
    at InflateRaw.afterTransform (_stream_transform.js:90:3)
```

**Comment by jsastriawan** [comment]
Thank you for your input. I realized hash was never been defined. I believe due to refactoring the code is moved into pass.js.

The fix to this issue is to replace obj.parent.hash with require('./pass').hash.

**Comment by Ylianst** [comment]
Oh! You are completely right. I just fixed it, MeshCentral v0.1.4-k just published. Thanks for that.

Let me know if you find more. Very appreciated.
Ylian
________________________________

**Comment by LinkPhoenix** [comment]
Hi Ylian,

We tested Meshcentral in my company, and it took us a long time to understand that the problem came from here. If we encounter other problems, I open another issue here.

Merci,

**Comment by LinkPhoenix** [comment]
I confirm, the problem is solved with new release v0.1.4-k

thank you very much
